transformer/task.py

```diff
@@ -212,7 +212,7 @@ def req_to_expr(r: Request) -> py.FunctionCall:
         if r.post_data:
             rpd = RequestsPostData.from_har_post_data(r.post_data)
             args.update(rpd.as_kwargs())
-    elif r.method is HttpMethod.PUT:
+    elif r.method in (HttpMethod.PUT, HttpMethod.PATCH):
```

Comment:
This should also be added to the equivalent block in the `lreq_to_expr` function (line 231).

Reply:
Done !